code/controllers/configuration.dm

```diff
@@ -531,6 +535,15 @@ var/list/storyteller_cache = list()
 				if("comms_password")
 					config.comms_password = value
 
+				if("forbidden_versions")
+					config.forbidden_versions = splittext(value, ";")
+
+				if("minimum_byond_version")
+					config.minimum_byond_version = value
```

Comment:
You'll want to slap text2num on both of these:

`= text2num(value)`

Reply:
Oh, thanks for this notification. I did indeed saw others values. I am not used with BYOND DM coding.